src/full/Agda/Interaction/Highlighting/LaTeX/Base.hs

```diff
@@ -768,7 +769,10 @@ generateLaTeXIO opts i = do
     BS.writeFile outPath latex
 
 latexOutRelativeFilePath :: TopLevelModuleName -> FilePath
-latexOutRelativeFilePath m = List.intercalate [pathSeparator] (List1.toList $ moduleNameParts m) <.> "tex"
+latexOutRelativeFilePath m =
+  List.intercalate [pathSeparator]
```

Comment:
`List.intercalate [pathSeparator]` is `System.FilePath.joinPath`.

Reply:
That is neither related to the pull request nor true (try `System.FilePath.Windows.joinPath ["C:\\", "C:\\"]`).